extensions/kafka-client/runtime/src/main/java/io/quarkus/kafka/client/serde/JsonbSerde.java

```diff
@@ -0,0 +1,122 @@
+package io.quarkus.kafka.client.serde;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serializer;
+
+/**
+ * A {@link Serde} that (de-)serializes JSON using JSON-B.
+ */
+public class JsonbSerde<T> implements Serde<T> {
+
+    private final Class<T> type;
+    private final Jsonb jsonb;
+
+    public JsonbSerde(Class<T> type) {
+        this(type, JsonbBuilder.create());
```

Comment:
I'm not an expert of JSONB. Should this instance be closed at some point?

Reply:
It is closed in the `close()` method below.